src/lib/journey/stages/generic.svelte

```diff
@@ -119,6 +122,7 @@
       props={{
         callback,
         callbackMetadata: metadata?.callbacks[idx],
+        journey: currentJourney,
```

Comment:
If we use the `journey` stack store directly in the callback component, then you won't have to worry about passing this value through the layers, cleaning up the code a bit.